FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Tasks/TaskExecutor.cs

```diff
@@ -488,6 +491,8 @@ private void AddFhirResourcesToCache(List<JObject> fhirResources, CacheResult ca
                                 skippedCount);
                         }
 
+                        taskContext.OutputCount =
+                            taskContext.OutputCount.AddToDictionary(resourceType, parquetStream.BatchSize);
```

Comment:
Nit: Can you help to rename the **BatchSize** in **StreamBatchData** to another like **BatchCount**? Now we have the output size, feel that name may leading to misunderstand.